src/components/flatParameters/FlatParameters.tsx

```diff
@@ -264,7 +267,13 @@ export function FlatParameters({
         const fieldValue = mixInitAndDefault(param);
         switch (param.type) {
             case 'BOOLEAN':
-                return <Switch checked={!!fieldValue} onChange={(e) => onFieldChange(e.target.checked, param)} />;
+                return (
+                    <Switch
+                        checked={!!fieldValue}
+                        onChange={(e) => onFieldChange(e.target.checked, param)}
+                        disabled={disabledParams !== undefined && disabledParams.indexOf(param.name) >= 0}
```

Comment:
```suggestion
                        disabled={disabledParams?.indexOf(param.name) >= 0}
```
It is better to use JS syntax..

Reply:
it doesn't work !